core/server/worker/src/main/java/alluxio/worker/grpc/AbstractReadHandler.java

```diff
@@ -111,14 +112,16 @@ public void onNext(alluxio.grpc.ReadRequest request) {
       mDataReaderExecutor.submit(createDataReader(mContext, mResponseObserver));
       mContext.setDataReaderActive(true);
     } catch (Exception e) {
+      LOG.warn("Exception occurred while processing read request {}.", request, e);
```

Comment:
This will dump the entire stack trace making it look like an error.

Reply:
Updated.